### ISSUE TITLE
TypeError: arg.parse is not a function when using `artillery run-test`

### ISSUE THREAD

**Issue by tchambers97** [artilleryio/artillery#2009]
Hi! I've been away for a while and I'm refreshing our artillery versions. I realize pro is being deprecated, and we're going to start switching to `run-fargate` instead, but as an interim step I'm trying to run version 4.0.0-1 with our existing tests.

Version info: 
```
Artillery 2.0.0-35
Artillery Pro 4.0.0-1
```

Running this command: 
```
> artillery run-test --output test-results/log__`date +%Y-%m-%d__%H-%M`.json --launch-type ecs:fargate --region us-east-1 --dotenv ../.env --cluster mycluster --subnet-ids subnet1,subnet2,subnet3 --environment qa --count 1 --config commerce/commerce-config-smoke.yml commerce/fetch-available-items.yml

```

I expected to see this happen:

Pro test execution

Instead, this happened:

TypeError: arg.parse is not a function

**Comment by hassy** [comment]
Hi @tchambers97 👋 It looks like there may be an issue with Artillery Pro v4 and the most recent release of Artillery. I'd suggest either:

* Rolling back to the most recent v3 release of Artillery Pro - `v3.0.0-24`
* Using new built-in support for running tests on Fargate. You'd need to tweak your existing Artillery command like so:

```sh
artillery run:fargate --output test-results/log__`date +%Y-%m-%d__%H-%M`.json --region us-east-1 --dotenv ../.env --cluster mycluster --subnet-ids subnet1,subnet2,subnet3 --environment qa --count 1 --config commerce/commerce-config-smoke.yml commerce/fetch-available-items.yml
```

The only changes are using `run:fargate` instead of `run-test` and dropping the `--launch-type` flag as that's the default now.

Let me know if that works!

**Comment by tchambers97** [comment]
Hi Hassy! Thanks for the quick response. I think switching to the Fargate command will work. It looks like I have some permission stuff to work out with our infrax team, but that's on our end. Thanks!

**Comment by bernardobridge** [comment]
This should be fixed in Artillery Pro 4.0.1. That being said, please move to the Open Source `run-fargate` command, as the legacy Artillery Pro is on maintenance mode, and will eventually be deprecated.

Docs:
- https://www.artillery.io/docs/reference/cli/run-fargate
- https://www.artillery.io/docs/load-testing-at-scale/aws-fargate
